### PR TITLE
RFR PR Issue Fix

### LINKED ISSUE #61: give Koala itself access to add reactions
When overwriting channel perms for rfr

### PR DESCRIPTION
**Pull request by Unknownlocal**

### CODE REVIEW

**Comment by Unknownlocal** [comment]
Fixes #61

**Comment by JayDwee** [comment]
Pull Master into this branch first

File: cogs/ReactForRole.py [hunk 1]
```diff
@@ -689,9 +689,10 @@ async def overwrite_channel_add_reaction_perms(self, guild: discord.Guild, chann
         overwrite: discord.PermissionOverwrite = discord.PermissionOverwrite()
         overwrite.update(add_reactions=False)
         await channel.set_permissions(role, overwrite=overwrite)
-        bot_members: List[discord.Member] = [member for member in guild.members if member.bot]
-        for bot in bot_members:
-            await channel.set_permissions(bot, overwrite=None)
+        bot_members = [member for member in guild.members if member.bot and "Koala" in str(member)]
```
Comment (JayDwee) [inline cogs/ReactForRole.py:692]: checking for containing Koala isn't the best way and can be exploiited, maybe try something like member.bot and member.id == self.bot.id or something like that

**Review by JayDwee — CHANGES_REQUESTED** [review]

**Review by JayDwee — CHANGES_REQUESTED** [review]
The bot still doesn't get perms, it just adds itself to the list with no changes to perms

image

**Review by JayDwee — APPROVED** [review]
Looks Good

**Pull request merged**